src/main/java/core/basesyntax/MainApp.java

```diff
@@ -1,5 +1,17 @@
 package core.basesyntax;
 
 public class MainApp {
+    public static void main(String[] args) {
 
+        Machine truck = new Truck();
+        Machine bulldozer = new Bulldozer();
+        Machine excavator = new Excavator();
+
+        String[] matrixMachine = {truck.goWork(),bulldozer.goWork(),excavator.goWork(),
+                truck.stopWork(), bulldozer.stopWork(), excavator.stopWork()};
+
+        for (String matrix : matrixMachine) {
```

Comment:
The variable name `matrixMachine` is not informative. It's better to use a name that reflects the data it holds, such as `machineStates` or `machineActions`.